textile/features.textile

```diff
@@ -1141,7 +1141,8 @@ h3(#activation-state-machine). Activation State Machine
 * @(RSH3)@ In platforms that support receiving push notifications, in order to connect the device's push features with Ably's, the library must perform the process described in the following abstract state machine. While this process should be implemented in whatever way better fits the concrete platform, it should be taken into account that its lifetime is that of the _app_ that runs it, which outlives that of the @RestClient@ instance or (typically) the process running the app. This typically forces some kind of on-disk storage to which the state machine's state must be persisted, so that it can be recovered later by new instances and processes running the app triggered by external events.
 ** @(RSH3a)@ State @NotActivated@ (the initial one).
 *** @(RSH3a1)@ On event @CalledDeactivate@:
-**** @(RSH3a1a)@ Makes @Push#deactivate@ return or call its callback with no error.
```

Comment:
We don't just delete items. Instead you should mark it "This spec was valid up until version...".

Reply:
See https://github.com/ably/specification/blob/main/CONTRIBUTING.md#features-spec-points